calculator/main.py

```diff
@@ -58,18 +76,22 @@ def get_history(calc):
         ie:
         ('2016-05-20 12:00:00', 'add', (1, 2), 3),
     """
-    pass
+    return calc['history']
+    
 
 
 def reset_history(calc):
     """
     Resets the calculator history back to an empty list.
     """
-    pass
+    calc['history']=[]
```

Comment:
Separate assignation from value with spaces for better formatting (https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements)

```python
calc['history'] = []
```